src/scripts.js

```diff
@@ -57,13 +61,65 @@ function gatherData() {
   Promise.all([getData('users'), getData('ingredients'),
     getData('recipes')])
     .then(data => {
-      const allUsers = data[0].usersData;
-      const allIngredients = data[1].ingredientsData;
-      const allRecipes = data[2].recipeData;
-      generateUser(allUsers);
-      createCards(allRecipes);
+      generateUserData(data[0]);
```

Comment:
This is really clean. I like passing the information directly to the functions that need it and getting rid of the globals.